### DISCUSSION TITLE
Icon needed for navigation not loading

### DISCUSSION THREAD

**Discussion by johannaweintraub** [twilio-labs/paste#763] [General] [answered]
[streamline-icon-wall-socket-1@140x140.svg.zip](https://github.com/twilio-labs/paste/files/5234089/streamline-icon-wall-socket-1%40140x140.svg.zip)

Could this icon please be added to Paste? Mid-October is the required date. 

https://share.goabstract.com/920337aa-b45a-4b5e-b67d-2336a69f1617

**Comment by SiTaggart** [comment]
Sure, we can add this to the backlog. Thanks for adding the delivery date requirement and placing the request well in advance 🙇 

Another option is always to contribute to Paste. There are guidelines on how to add icons to the system https://paste.twilio.design/icons/how-to-add-an-icon

**Comment by corinne-nagel** [comment]
Hey @johannaweintraub, is this icon planned on being used anywhere else?

**Comment by johannaweintraub** [comment, marked as answer]
Not at the moment

**Reply by corinne-nagel** [reply]
Since this wouldn't be an icon used in other places we wouldn't suggest adding it to Paste; we ideally are only adding systemized icons. If you think you need to use the light switch icon, we may suggest you just doing a one-off addition yourself through this: https://github.com/twilio-labs/svg-to-react

After looking at the full design, the team thinks it might make more sense to add the error icon in this state instead of adding an entirely new icon. The visual cue of using the same error icon will help users relate the alert above to the error in the navigation. Also, if you could match the styles you're using with the empty state shown here to keep our patterns consistent, that would be great! https://share.goabstract.com/b72d29bc-9dbf-4f99-9a52-d3b4e6bf53cd?collectionLayerId=1a0fdec9-70cc-4303-93c9-6e3495196b94&mode=design

Hope this helps!

**Comment by johannaweintraub** [comment]
Ok, we will use the grey colored error icon as we don't want to alarm users
with the red. I agree the use of that icon will help tie it back to the
alert. I will update the empty state to white. Sorry, I was looking at an
old spec I believe. You can close this out. Thanks
